migrations/event_rtype_migration_test.go

```diff
@@ -0,0 +1,133 @@
+package migrations
+
+import (
+	"testing"
+	"time"
+
+	"github.com/evergreen-ci/evergreen/db"
+	"github.com/evergreen-ci/evergreen/model/event"
+	"github.com/mongodb/anser"
+	anserdb "github.com/mongodb/anser/db"
+	"github.com/stretchr/testify/suite"
+	"gopkg.in/mgo.v2/bson"
+)
+
+type eventRTypeMigrationSuite struct {
+	events []anserdb.Document
+
+	migrationSuite
+}
+
+func TestEventRTypeMigration(t *testing.T) {
+	suite.Run(t, &eventRTypeMigrationSuite{})
+}
+
+func (s *eventRTypeMigrationSuite) SetupTest() {
+	loc, err := time.LoadLocation("UTC")
+	s.NoError(err)
+	date, err := time.ParseInLocation(time.RFC3339Nano, "2017-06-20T18:07:24.991Z", loc)
+	s.NoError(err)
+
+	s.NoError(db.ClearCollections(event.AllLogCollection))
```

Comment:
I'd avoid using the evergreen db package if possible.

Reply:
What is your suggestion for clearing collections in anser tests then?

Reply:
I used  the anser DB session directly with RemoveAll, unless you've got a better suggestion